decimal.go

```diff
@@ -113,9 +113,13 @@ func NewFromString(value string) (Decimal, error) {
 		expInt, err := strconv.ParseInt(value[eIndex+1:], 10, 32)
 		if err != nil {
 			if e, ok := err.(*strconv.NumError); ok && e.Err == strconv.ErrRange {
-				return Decimal{}, fmt.Errorf("can't convert %s to decimal: fractional part too long", value)
+				return Decimal{}, &ErrorFractionLimit{value: value}
```

Comment:
Do you think it is better to call it `ErrorExponentLimit`? The documentation mentions 'exponent' but not 'fraction'.

Reply:
Done